contracts/MyToken.sol

```diff
@@ -6,7 +6,10 @@ import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
 contract MyToken is ERC20 {
     uint8 private _decimal = 5;
+    uint256 private allocate_token = 1000;
```

Comment:
you'll need to adjust this value later. An allocated token value of 1000 and a decimal of 5 means that you have created 1000/10^5 = 0.01 DTN.